Source/WebCore/bindings/scripts/test/TestObj.idl

```diff
@@ -124,12 +124,9 @@ enum TestConfidence { "high", "kinda-low" };
     [ReflectSetter] attribute boolean reflectedSetterBooleanAttr;
     [ReflectSetter] attribute Element reflectedSetterElementAttr;
     [ReflectSetter] attribute FrozenArray<Element> reflectedSetterElementsArrayAttr;
-    [ReflectSetter, URL] attribute DOMString reflectedSetterURLAttr;
```

Comment:
Per spec you can't do ReflectSetter and ReflectURL together so this just removes the related test cases (this is never used in practice anyway)